Kconfig.zephyr

```diff
@@ -235,6 +235,12 @@ config NO_OPTIMIZATIONS
 
 endchoice
 
+config COMPILER_COLOR_DIAGNOSTICS
+	bool "Disable colored diganostics"
```

Comment:
I belief this should be Enable instead of Disable?

Reply:
Fixed.